QUEST_JOBSTEP.tsv

```diff
@@ -3013,118 +3013,118 @@ QUEST_JOBSTEP_20150729_003012
+QUEST_JOBSTEP_20150730_003064	The world knows me as Squire Master.{nl}But, I am also the sole survivor of the Legwyn Family.{nl}
+QUEST_JOBSTEP_20150730_003065	As the last head of the family, I am trying to raise them once more.{nl}If you could find the seal from the mansion of the Legwyn Family, I will teach you everything about a Squire.
+QUEST_JOBSTEP_20150730_003066	It's okay if it's not intact.{nl}If we have the seal of the family, we can let the rest of the world know that the Legwyn Family is still alive.
+QUEST_JOBSTEP_20150730_003067	So horrible...{nl}If I become the official heir of the Legwyn Family, I will never forgive.
+QUEST_JOBSTEP_20150730_003068	Of course, I should return your favor first as a Squire Master.
+QUEST_JOBSTEP_20150730_003069	This was when the Coral Plains was still a sea.{nl}I once fought with the great pirate, Bradislovas for the throne of ocean.{nl}
+QUEST_JOBSTEP_20150730_003070	I won at that time, but I lost his necklace which I obtained after beating him into the sea.{nl}The sea was very deep back then, but I may able to find it now.
+QUEST_JOBSTEP_20150730_003071	The people who said my words were bluffs would be surprised if I could find it.{nl}It must be at around 32_1 of the Coral Plains. You are the Corsair that I acknowledged so you will find it easily.  
 QUEST_JOBSTEP_20150730_003072	Good!{nl}I will teach you everything. 
-QUEST_JOBSTEP_20150730_003073	Well done. It could have been better if we were to find it little sooner.{nl}I will teach you the new formation as a return. It's precise so it's more strong.
-QUEST_JOBSTEP_20150730_003074	If you are Doppelsoeldner, you should something for the kingdom.{nl}I want to do it myself, but since it is being done quietly, I will be spotted if I do it myself.
-QUEST_JOBSTEP_20150730_003075	Thanks for saying so. Please find the envoy of Ramin General secretly.{nl}His last contact was from 36_2 of Pilgrim's way.
-QUEST_JOBSTEP_20150730_003076	Sad. I guess he was attacked by the monsters.{nl}Anyways, since you've become a member of Doppensoeldners, I will support you with my best.
-QUEST_JOBSTEP_20150730_003077	Ah, and please keep this secret to other people.{nl}I gotta find an other way to contact Ramin general.
-QUEST_JOBSTEP_20150730_003078	If you've explored the area completely in the past, just record it in the orb.{nl}After exploring all the areas in Crystal Mine, use the orb to record your past.
-QUEST_JOBSTEP_20150730_003079	Explore all the regions in the 1st and 2nd districts with this orb.{nl}If you've explored the areas completely in the past, just record it in the orb.{nl}After exploring all the area, use the orb to record your past.
+QUEST_JOBSTEP_20150730_003073	Well done. It would have been better if we found it little sooner.{nl}However, I will teach you the new formation as a return. It's precise so it's stronger.
```

Comment:
"new formation in return"
instead of
"new formation as a return"
Would work here.